internal/service/serviceinstall.go

```diff
@@ -37,13 +37,33 @@ func Register(ctx *log.Context) error {
 	if !isSystemdSupported(ctx) {
 		return errors.New("Systemd not supported. Failed to register service")
 	}
+	targetVersion := os.Getenv("AZURE_GUEST_AGENT_EXTENSION_VERSION")
```

Comment:
Move this to constant and also add link to agent doc that explains about the env variable.

Reply:
Fixed :)